### DISCUSSION TITLE
Could you also make a customizer.conf.j2 with a location /push/

### DISCUSSION THREAD

**Discussion by jflipsen** [HCL-TECH-SOFTWARE/connections-automation#10] [General]
In https://help.hcltechsw.com/connections/v7/admin/install/inst_post_nginx.html?hl=nginx
is described to make a nginx conf for longpolling, but there is no combi conf for customizer and longpolling.

My main concern are the headers.  do they merge / inherent all the headers of the rest of the nginx conf for location /push/?

**Comment by dejo1307** [comment]
Actually great suggestion. Yes, I just opened the issue: https://github.com/HCL-TECH-SOFTWARE/connections-automation/issues/11

Since everyone is currently vacationing, will be done as soon as we are back. Will be also tracked here in board.
